layouts/partials/head.html

```diff
@@ -30,16 +31,28 @@
   <title>{{ .Site.Title }} | {{ replace .Site.Params.Owner "%Vote.gov%" "Vote.gov" }}</title>
   <meta name="description" content="{{ .Site.Params.meta__description }}"/>
   <meta property="og:title" content="{{ .Site.Params.meta__title }}"/>
+  <meta property="twitter:title" content="{{ .Site.Params.meta__title }}"/>
+
   <meta property="og:description" content="{{ .Site.Params.meta__description }}"/>
+  <meta property="twitter:description" content="{{ .Site.Params.meta__description }}"/>
+  
   {{ else }}
 <!--  NODE -->
   <title>{{ replace $translation.register.state_heading "%state_name%" .Params.state_name }} | {{ replace .Site.Params.Owner "%Vote.gov%" "Vote.gov" }}</title>
   <meta name="description" content="{{ replace $translation.register.meta_name "%state_name%" .Params.state_name }}" />
   <meta property="og:title" content="{{ replace $translation.register.state_heading "%state_name%" .Params.state_name }}" />
+  <meta property="twitter:title" content="{{ replace $translation.register.state_heading "%state_name%" .Params.state_name }}" />
+
   <meta property="og:description" content="{{ replace $translation.register.meta_name "%state_name%" .Params.state_name }}" />
+  <meta property="twitter:description" content="{{ replace $translation.register.meta_name "%state_name%" .Params.state_name }}" />
   {{ end }}
+  <meta property="og:type" content="website"/>
   <meta property="og:url" content="{{ .Permalink }}"/>
```

Comment:
This `og:url` needs to be an absolute url. Add `https://vote.gov` before `{{ .Permalink }}`
Additionally we need to update the canonical link. 
`<link rel="canonical" href="{{ .Permalink }}"/>` Add `https://vote.gov` before `{{ .Permalink }}`